src/site/content/en/fast/codelab-imgix/index.md

```diff
@@ -0,0 +1,121 @@
+
+## imgix URLs
+
+Once your source is deployed (this usually happens immediately but can sometimes take longer), you can start serving your images via this custom domain name, for example `example.imgix.net/desk.jpg`. The structure of a basic imgix URL follows this format:
+
+![https://example.imgix.net/products/desk.jpg?w=600&exp=1](./imgix-url.png)
+
+In the above URL, the name of the Source is `example`, so the hostname takes the form of `example.imgix.net`. The path consists of any additional directory information required to locate your image within your image storage (for example if you have different subfolders for your images). In this example, `/products/desk.jpg` completes the full path to the image. Once you are able to access your master image from the origin source, you can use imgix's API via query parameters. In the above example, `w=600` resizes the image to 600 pixels wide while `exp=1` increases the image's exposure by 1 unit.
+
+## imgix client libraries
+
+After you become familiar with how to form imgix URLs, you can begin to integrate them in your website. However, doing this manually can take a lot of time, especially depending on the size of the application. To facilitate this, imgix offers a variety of [client libraries](https://docs.imgix.com/libraries?utm_source=webdev&utm_medium=referral&utm_campaign=codelab), which allows developers to programmatically build URLs and streamline integration.
+The imgix SDK offers both backend and frontend libraries in a variety of common languages and web frameworks, each providing utilities that follow imgix's recommended best practices. This codelab will explain how to use the imgix.js library.
+
+## Serve responsive images using imgix.js
+
+[imgix.js](https://github.com/imgix/imgix.js) is a Javascript library that allows developers to easily generate responsive images using the `srcset` and `sizes` attributes. What usually requires a lot of manual configuration on the frontend can now be done in just a few lines code.
+
+### Getting started
+
+For the purposes of this Glitch, imgix.js is already installed. For more information on installation and usage, visit the [imgix.js github](https://github.com/imgix/imgix.js).
+
+### View this demo
+
+- To view this demo press the _View App_ button, then the _Expand_ button ![Expand button](./expand-button.png)
```

Comment:
I replaced this with the Instruction component.

```
{% Instruction 'preview' %}
```

It's useful to use this widget because when Glitch changes their UI (which happens a lot) we can update the text in the widget and all of our docs will get the change. fyi @kaycebasques

Reply:
Here's a link to some examples: https://web.dev/handbook/web-dev-components/#instruction

There are a decent number of instructions already in the component.
https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/Instruction.js